metals/src/main/scala/scala/meta/internal/metals/debug/DebugParametersAdapter.scala

```diff
@@ -0,0 +1,47 @@
+package scala.meta.internal.metals.debug
+
+import ch.epfl.scala.{bsp4j => b}
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.meta.internal.debug.{protocol => jvm}
+import scala.meta.internal.metals.BuildTargets
+import scala.meta.internal.metals.CompilationScheduler
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.debug.protocol.LaunchParameters
+import scala.meta.internal.metals.debug.{protocol => metals}
```

Comment:
Let's avoid renaming packages into `metals`. By convention in this codebase, we there are two namespaces with conflicting names like `scala.meta.Position` and `org.eclipse.lsp4j.Position` then we use single character renames

```scala
import scala.{meta => m}
import org.eclipse.{lsp4j => l}
// m.Position
// l.Position
```